linetools/spectra/lsf.py

```diff
@@ -213,6 +230,143 @@ def load_COS_data(self):
         
         return pixel_scale, data
 
+    def load_STIS_data(self):
+        """Load the right data according to `instr_config` for HST/STIS
+        instrument"""
+
+        # define pixel scales; values obtained from STScI
+        # these values must be consistent with the given LSFs
+        pixel_scale_dict = {
+            'G140L': 0.60 * u.AA,
+            'G140M': 0.05 * u.AA,
+            'G230L': 1.58 * u.AA,
+            'G230M': 0.09 * u.AA,
+            'E140M': 'lambda/91700',
+            'E140H': 'lambda/228000',
+            'E230M': 'lambda/60000',
+            'E230H': 'lambda/228000',
+            'G230LB': 1.35 * u.AA,
+            'G230MB': 0.15 * u.AA,
+            'G430L': 2.73 * u.AA,
+            'G430M': 0.28 * u.AA,
+            'G750L': 4.92 * u.AA,
+            'G750M': 0.56 * u.AA
+        }
+        # define channel based on grating name
+        channel_dict = {
+            'G140L': 'FUV-MAMA',
+            'G140M': 'FUV-MAMA',
+            'G230L': 'NUV-MAMA',
+            'G230M': 'NUV_MAMA',
+            'E140M': 'FUV-MAMA',
+            'E140H': 'FUV-MAMA',
+            'E230M': 'NUV-MAMA',
+            'E230H': 'NUV-MAMA',
+            'G230LB': 'CCD',
+            'G230MB': 'CCD',
+            'G430L': 'CCD',
+            'G430M': 'CCD',
+            'G750L': 'CCD',
+            'G750M': 'CCD'
+        }
+        # also need slits
+        available_slits = {
+            'G140L': ['52x0.1', '52x0.2', '52x0.5', '52x2.0'],
```

Comment:
I'm getting a couple of errors when trying to extract LSFs for a couple different gratings.  For the G140L, I tried:
```
lf1 = LSF(dict(name='STIS', grating='G140L',slit='52x0.2'))
fun=lf1.interpolate_to_wv0(1500.*u.AA)
```
but I get the following error:

/Users/burchett/Dropbox/Code/linetools/linetools/spectra/lsf.py in interpolate_to_wv0(self, wv0)
    423             # we don't want to extrapolate wildly, but allow LSF instantiations for wv0 outside range of 'col_waves'
    424             if (wv0 >= col_waves[0]) & (wv0 <= col_waves[-1]):
--> 425                 f = interp1d(col_waves_aux,aux_val,bounds_error=True,kind='linear')  # no need to extrapolate
    426                 lsf_vals += [f(wv0)]
    427 

/Users/burchett/Applications/anaconda/lib/python2.7/site-packages/scipy/interpolate/interpolate.pyc in __init__(***failed resolving arguments***)
    479         if len(self.x) < minval:
    480             raise ValueError("x and y arrays must have at "
--> 481                              "least %d entries" % minval)
    482 
    483     @property

ValueError: x and y arrays must have at least 2 entries